masonry/src/passes/accessibility.rs

```diff
@@ -111,6 +111,10 @@ fn build_access_node(widget: &dyn Widget, state: &WidgetState, scale_factor: f64
     if state.clip.is_some() {
         node.set_clips_children();
     }
+    if state.in_focus_chain {
+        node.add_action(accesskit::Action::Focus);
+        node.add_action(accesskit::Action::Blur);
```

Comment:
I presume it's meaningful to have the blur action available on an already unfocused widget?

Reply:
No, it's not. But can I check whether the widget is focused without passing global state into this function?

Reply:
As far as I can see, the global state is available in the calling context, so passing it in is fine.
I'd probably shift this code around so that the functionality to set all of these properties is passed the `AccessCtx`.

Reply:
OK, I think I just did what you suggested.